relaxed.module

```diff
@@ -96,5 +96,5 @@ function multiversion_entity_create_access(AccountInterface $account, array $con
  * Implements hook_entity_field_access().
  */
 function relaxed_entity_field_access($operation, FieldDefinitionInterface $field_definition, AccountInterface $account, FieldItemListInterface $items = NULL) {
-  return AccessResult::allowedIfHasPermission($account, 'perform content replication');
+  return AccessResult::allowedIfHasPermissions($account, ['perform push replication', 'perform pull replication'], 'OR');
```

Comment:
We need to differentiate on $operation. Pull replication should only have access to read/view fields. And push replication should have edit/delete also.